specification/network/resource-manager/Microsoft.Network/preview/2022-04-01-preview/networkManagerGroup.json

```diff
@@ -573,6 +582,16 @@
         "resourceId": {
           "type": "string",
           "description": "Resource Id."
+        },
+        "region": {
```

Comment:
Why we need region for static member

Reply:
this was requested for portal purposes; we want to be able to display cross-tenant vnet regions